bin/tick.sh

```diff
@@ -6,6 +6,9 @@
 
 # NeoGo binary path.
 NEOGO="${NEOGO:-docker exec -it main_chain neo-go}"
```

Comment:
`invokefunction` can use interactive neo-go invocation because of `expect`?

Reply:
Yes, exactly